identity-service/src/routes/healthCheck.js

```diff
@@ -4,11 +4,91 @@ const { handleResponse, successResponse, errorResponseServerError } = require('.
 const { sequelize } = require('../models')
 const { getRelayerFunds } = require('../txRelay')
 const Web3 = require('web3')
```

Comment:
i don't think we need this Web3

Reply:
it's used in `/balance_check` endpoint :)

Reply:
awesome! thanks for checking!